src/Service/SF1500/SF1500.php

```diff
@@ -61,10 +59,10 @@ public function getPersonName(string $brugerId): string
             'ns2UUIDIdentifikator',
         ];
 
-        $personId = $this->getValue($data, $personIdKeys, '');
+        $personId = $this->getValue($data, $personIdKeys);
 
         if (null === $personId) {
-            return '';
+            throw new SF1500Exception('Could not find person id.');
```

Comment:
This may be a matter of (personal) taste, but I think “Cannot” is more correct than “Could not” in exception messages.

Reply:
Fixed.